src/Company_Coding_Round_Questions/Find_PIN.java

```diff
@@ -0,0 +1,121 @@
+package Company_Coding_Round_Questions;
```

Comment:
Please add here the problem statement

Reply:
I have added the problem statement in the code